languages/thingtalk/en/filters.genie

```diff
@@ -482,3 +482,8 @@ whose_npp_filter = {
     p:out_param_Any 'is' x:constant_Any => C.makeFilter(p, 'contains', x);
 }
 
+
+//FIXME: calculate threshold using max rating instead of a fixed number
+quality_filter = {
+    ('good' | 'great' | 'nice' | 'decent') => C.makeFilter(new Ast.Value.VarRef('aggregateRating.ratingValue'), '>=', new Ast.Value.Number(4));
```

Comment:
This is really weird to have in the domain-independent templates. It uses a schema.org specific parameter, and a schema.org specific value. I'd rather have webqa-process-schemaorg make up some dataset.tt automatically.

Reply:
the problem is that we do not have a way to apply "good restaurant" in dataset.tt to generate "good chinese restaurant", we need to update how we use primitive templates for that ...

Reply:
We really need domain-specific templates with the full (or close to full) power of Genie templates... But that's a matter for another day.